debian/tests/docker-in-lxd

```diff
@@ -104,7 +104,14 @@ if [ -n "${noproxy:-}" ]; then
     lxc config set core.proxy_ignore_hosts $noproxy
 fi
 
-lxc launch ubuntu-daily:${suite}/${arch} docker -c security.nesting=true -c security.privileged=true
+# Follow the official tutorial on how to run docker inside lxd containers
+# https://ubuntu.com/tutorials/how-to-run-docker-inside-lxd-containers
+lxc storage create docker-storage btrfs
+lxc launch ubuntu-daily:${suite}/${arch} docker
+lxc storage volume create docker-storage docker-vol
+lxc storage volume attach docker-storage docker-vol docker /var/lib/docker
+lxc config set docker security.nesting=true security.syscalls.intercept.mknod=true security.syscalls.intercept.setxattr=true
+lxc restart docker
```

Comment:
I believe you can save this extra restart by creating the container already with the security.settings. The full steps would be:
```
# Follow the official tutorial on how to run docker inside lxd containers
# https://ubuntu.com/tutorials/how-to-run-docker-inside-lxd-containers
lxc storage create docker-storage btrfs
lxc storage volume create docker-storage docker-vol
lxc launch ubuntu-daily:${suite}/${arch} docker -c security.nesting=true -c security.syscalls.intercept.mknod=true -c security.syscalls.intercept.setxattr=true
lxc storage volume attach docker-storage docker-vol docker /var/lib/docker
```
I tried the above and could then proceed to install docker and run the hello-world container inside that lxd on a lunar image.